src/libraries/System.IO.Packaging/src/System/IO/Packaging/FileFormatException.cs

```diff
@@ -55,7 +55,7 @@ public FileFormatException(Uri? sourceUri)
             : base(
                 sourceUri == null
                 ? SR.FileFormatException
-                : SR.Format(SR.FileFormatExceptionWithFileName, sourceUri))
+                : SR.FileFormatExceptionWithFileName)
```

Comment:
this is a case where the resource is clearly wrong. both these resources are the same string.

probably the right fix for these two is to change the resource to eg `File "{0}" has an invalid file format.` or similar, and leave the sourceUri in it.